### DISCUSSION TITLE
OpenSSL 1.0.2t and FIPS support

### DISCUSSION THREAD

**Discussion by dayapatil** [nodejs/help#4833] [Q&A] [answered]
Any plans to support OpenSSL 1.0.2t(FIPS supported) release in Node.JS v14.X or onwards?

**Comment by mhdawson** [comment, marked as answer]
8.x was the last version that supported the 1.0.2 series. At this point all streams use the 1.1.1 series. 

There are people looking how we integrate the 3 once its available and that is the next one that "may" have public FIPS support that we can incorporate.  

Some linux distros build Node.js differently and can provide FIPS support but that requires changes to the code and there is no option to do that from the code in the public repo today.
